docs/products/platform/accounts/guides/user-permissions/index.md

```diff
@@ -1,39 +1,67 @@
-    - **Read Only**: The user can view the resource and all of its associated information typically visible within the Linode Manager, however they can not otherwise interact with it through Cloud Manager.
-    - **Read-Write**: The user has full access to the selected resource, and can make any changes that only an administrator is otherwise able to. This includes resource deletion, cloning, and all other applicable edits.
+1. When you have finished configuring the user's permissions, click **Save**. The user's permissions are saved and become effective immediately.
 
-1. When you have finished configuring the user's permissions, click **Save**. The user's permissions are saved and effective immediately.
\ No newline at end of file
+## Permission Settings
+
+### Full Account Access
+
+![Full Account Access](user-permissions-account-access.png)
+
+A user can either be given unrestricted (full) access to an account or they can be restricted by default. If **Full Account Access** is set to *ON*, the user has full access to the account and no further permission settings are available. When set to *OFF*, the user has no access and individual permissions can be granted using the other settings (covered below).
+
+### Create Services (Global Permissions)
+
+![Global Permissions](user-permissions-global-permissions.png)
+
+The **Global Permissions** settings enable users to add various types of services. There is a setting that corresponds to each type of service. For instance, to allow the user to add NodeBalancers, enable the *Can add NodeBalancers to this account* setting.
+
+{{< note >}}
+Granting access to settings denoted with a dollar sign ($) allows the user to perform actions that incur billing costs, such as adding or resizing a Linode Compute Instance.
+{{< /note >}}
+
+### Billing Access
+
+![Billing Access](user-permissions-billing-access.png)
+
+This section provides various levels of access to view or modify billing information:
+
+- **None**: The user is unable to view any billing information. This does not prevent a user from creating billable resources, which are instead applied as **Global Permissions** in the previous section.
+- **Read Only**: The user can [View Invoices](/docs/products/platform/billing/guides/view-history/) and [Access Billing Info](/docs/products/platform/billing/guides/access-billing/).
+- **Read-Write**: The user has full access to [Billing Information](/docs/products/platform/billing/guides/access-billing/), can make payments, edit billing information, view billing information, receive copies of all invoices, and receive email related to payments.
+
+### Specific Services
+
+![Specific Services](user-permissions-specific-access.png)
+
+A user can be granted individual permissions to each instance of a service, such as Linode Compute Instances, Block Storage Volumes, NodeBalancers, and more Unlike **Global Permissions**, **Specific Permissions** apply to individual resources -- not the service as a whole.
```

Comment:
I'm not sure if it's worth noting in this guide, but there may be room to make this distinction explicit:

We could include text that explains the difference between access to a resource in the Cloud Manager, vs access to the resource via SSH and other services running on the resource. We could add in some text about how revoking access to the Cloud Manager does not automatically revoke access to SSH, and vice versa. And possibly link to this guide: https://www.linode.com/docs/guides/create-limited-developer-account/

Reply:
Good call out. Just added a note below the first paragraph to address this:

{{< note >}}
The permissions discussed within this guide apply to accessing the Linode platform through the Cloud Manager, Linode API, and Linode CLI. If you wish to only provide someone with direct access to the internal system of a Compute Instance (or revoke their access), that can be accomplished through limited user accounts within your operating system. See [Create an Account for a Developer to Work on Your Compute Instance](/docs/guides/create-limited-developer-account/).
{{< /note >}}